File: app/src/main/java/com/example/milkaggregatorapplication/IApiService.java
package com.example.milkaggregatorapplication;

import com.example.milkaggregatorapplication.model.MsgObj;
import com.example.milkaggregatorapplication.model.RequestModel;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;
import retrofit2.http.Path;

public interface IApiService {
    @POST("{phoneNoId}/messages")
    Call<MsgObj> sendMessage(@Path("phoneNoId")String phoneNoId,
                             @Body RequestModel requestModel

                             );
}

File: app/src/main/java/com/example/milkaggregatorapplication/IApiService.kt
package com.example.milkaggregatorapplication

import retrofit2.http.POST
import com.example.milkaggregatorapplication.model.RequestModel
import com.example.milkaggregatorapplication.model.MsgObj
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Path

interface IApiService {
    @POST("{phoneNoId}/messages")
    fun sendMessage(
        @Path("phoneNoId") phoneNoId: String?,
        @Body requestModel: RequestModel?
    ): Call<MsgObj?>?
}